aggregator/elem.go

```diff
@@ -359,16 +359,16 @@ func NewTimerElem(id id.RawID, sp policy.StoragePolicy, aggTypes policy.Aggregat
 
 // ResetSetData overrides elemBase.ResetData, to include quantile update for timer elements.
 func (e *TimerElem) ResetSetData(id id.RawID, sp policy.StoragePolicy, aggTypes policy.AggregationTypes) {
+	e.elemBase.ResetSetData(id, sp, aggTypes)
+
 	if aggTypes.IsDefault() {
-		aggTypes = e.opts.DefaultTimerAggregationTypes()
+		e.aggOpts.ResetSetData(e.opts.DefaultTimerAggregationTypes())
```

Comment:
Put a TODO in there to ensure the aggTypes stored in elemBase agree with what we store in aggregation options?